stable/mongodb/0.4.7/chart/values.yaml

```diff
@@ -0,0 +1,74 @@
+_config:
+  _metadata:
+    name: "mongodb"
+    version: "0.4.7"
+    description: ""
+  mongodb:
+    containers:
+      mongodb:
+        image: "bitnami/mongodb:3.4.2-r0"
```

Comment:
If we provide images in private docker registry (cargo) for user, we should generate charts 
 dynamically while  deploying a new cluster rather than use existing ones. 

These charts will be tested in this release.